src/books.py

```diff
@@ -0,0 +1,28 @@
+from flask import Blueprint, jsonify
+from flasgger import swag_from
+
+from src.models import Book
+from src.constants.http_status_codes import HTTP_200_OK
+
+
+books_bp = Blueprint('books', __name__, url_prefix='/api')
+
+
+@books_bp.get("/books/available")
+@swag_from("./docs/books/available_books.yaml")
+def available_books():
+
+    books = Book.query.filter_by(is_available=True)
+
+    books_object = []
+
+    for book in books:
+        books_object.append({
+            "id": book.id,
+            "name": book.name,
+            "author": book.author.first_name + " " + book.author.last_name,
+            "category": book.category,
+            "isbn": book.isbn
+        })
+
+    return jsonify(books_object), HTTP_200_OK
```

Comment:
Swagger documentation mentions that this endpoint returns a 404 when no book is available. You may want to handle it in this function alongside.

Reply:
I was going to resolve it. It should return an empty list if no books are available